source/slang/slang-ast-builder.h

```diff
@@ -114,16 +114,21 @@ class ASTBuilder : public RefObject
     {
         union
         {
-            NodeBase* nodeOperand;
-            int64_t intOperand;
+            NodeBase* nodeOperand[2];
```

Comment:
Why are there 2 of these now?

Reply:
This isn't ideal, but we need these to be able to encode a `DeclRef`, which is currently containing two pointers.

After the DeclRef representation cleanup, it will be shrinked down to just one pointer so we can scale this back.